packaging/flavorgen/cloudprovider/cloud-controller-manager.go

```diff
@@ -35,295 +35,64 @@ const (
 	CPIControllerImageURL = "gcr.io/cloud-provider-vsphere/cpi/release/manager"
 )
 
-// CloudControllerManagerServiceAccount returns the ServiceAccount used for the cloud-controller-manager.
-func CloudControllerManagerServiceAccount() *corev1.ServiceAccount {
-	return &corev1.ServiceAccount{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "cloud-controller-manager",
-			Namespace: "kube-system",
-		},
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "ServiceAccount",
-			APIVersion: corev1.SchemeGroupVersion.String(),
-		},
+// CloudControllerManagerManifests returns the yaml manifests generated via
+// `helm template` from https://github.com/kubernetes/cloud-provider-vsphere/tree/master/charts/vsphere-cpi
+func CloudControllerManagerManifests() (string, error) {
+	objectList := []string{}
+
+	// Replace the hardcoded image tag by a variable for clusterctl. This way we can dynamically configure the verison of CPI.
```

Comment:
```suggestion
	// Replace the hardcoded image tag by a variable for clusterctl. This way we can dynamically configure the version of CPI.
```

I would have ignored this nit, but golangci-lint didn't ...